hedera-mirror-rest/__tests__/service/nftService.test.js

```diff
@@ -0,0 +1,75 @@
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+'use strict';
+
+const integrationDbOps = require('../integrationDbOps');
+const integrationDomainOps = require('../integrationDomainOps');
+const NFTModel = require('../../model/nft');
```

Comment:
nit, should switch to import from the new package exports:

```suggestion
const {NFTModel} = require('../../model');
```